docs/overview/terminology.md

```diff
@@ -1,34 +1,54 @@
+| **Contract Agreement**       | **Contract Agreement**<p>* points to a **Contract Offer**<p>* results from a **Contract Negotiation Process**<p>* has a start date and may have a expiry date and a cancellation date
+| **Contract Negotiation**     | * MVP: only possible to accept already offered contracts. Counter offers are rejected automatically.
+| **Contract Offer**           | **Contract offer**<p>* set of obligations and permissions<p>* generated on the fly on provider side (see **Contract Offer Framework**)<p>* are immutable<p>* persisted in **contract negotiation process** once the negotiation has started<p>
+| **Contract Offer Framework** | **Contract offer framework**<p>* generates **contract offer templates**<p>* provided by **extensions**<p>* may be implemented in custom **extensions** to created **contract offers** based on existing systems
+| **Contract Offer Template**  | Blueprint of a **contract offer**
+| **Consumer**                 | see **data consumer**
+| **Consumer Pull**            | The data transfer is initilazied by the Consumer
+| **Data**                     | **Data** describes the properties of business objects
+| **Data Consumer**            | **Data consumer** are receiver of data in a data transfer
+| **Data Provider**            | **Data providers** produce data and are the source for data in a data transfer
+| **DataAddress**              | Reference to the physical Address of an **asset**
+| **DataFlow**                 | Transfer of **data** from **data producer** to **data consumer**.
+| **Dataspace**                | A **dataspace** is a space where different participants exchange **data** to get an benifit of this
+| **EDC Extension**            | A module piece of the EDC which can be customized used for the EDC
+| **EndpointDataReference**    | **EndpointDataReference** describes an endpoint at which one receives data
+| **Event**                    | An **event** is a action which is be observed in the EDC and published by the **Event Framework**
+| **Event Framework**          | **The event framework** is a way to react to actions in the EDC and can also be extended
+| **Event Router**             | **Event router** is the central part of the **event framework**, which collects all published **events**
+| **Extension**                | see **EDC Extension**
+| **Federated Catalog**        | A decentralized **catalog**
+| **Framework**                | see **contract offer framework**
+| **Health**                   | Infrastructure status of the EDC
+| **HTTPDataAddress**          | Specific type of **dataaddress** in context of HTTP-endpoints
+| **Identity Provider**        | Management of the correct identification of all participants in the **dataspace**.
+| **Injection**                | **Artifacts**/**EDC Extensions** can be injected in different parts of the EDC to be used 
```

Comment:
That's just plain wrong.

Reply:
I deleted this also because of your advice in context of what has to be in javadoc and what in markdown.

Only my question about it is this. The Injection Interface is used to make the different modules usable among each other. So to give a module of the EDC the possibility to find the COde of another used module within the EDC. If this is a complete misconception of your injection mechanism, I need to look deeper into your injection mechanism.